docs/collecting-data/code-generation/working-with-gtm/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Using Snowtype with Google Tag Manager"
+sidebar_position: 6
+---
+
+:::info
+This feature is available from version 0.5.0.
+:::
+
+To make working with Google Tag Manager and event tracking easier, we created a specific target for Snowtype fitting the way Google Tag Manager handles custom JavaScript code.
```

Comment:
I would write few use cases/benefits (even as simple as bullet points) about this feature

Reply:
Done!

Reply:
I like it! Well done!